src/daemon/daemon.c

```diff
@@ -62,6 +61,8 @@
 #define NW_GET_DAEMON(nw) \
 	(g_object_get_data(G_OBJECT(nw), "_notify_daemon"))
 
+int exit_timeout_seconds = 30; // Can be set by the --timout option, see mnd-daemon.c for details
```

Comment:
Change from the `IDLE_SECONDS` macro to the `exit_timeout_seconds` int so it can be changed at runtime by the command line option.